### PR TITLE
Error mail resolved [ISSUE #2]

### LINKED ISSUE #2: Error when subscribing with an already chosen mail address
The bug speaks of itself.

While subscribing to the website, I’ve had a bug when I tried to subscribe an account with a mail address that belonged to another account.

Here is an excerpt of the Traceback:

```
    rec.execute(self)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/orm/unitofwork.py", line 556, in execute
    uow
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 181, in save_obj
    mapper, table, insert)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/orm/persistence.py", line 866, in _emit_insert_statements
    execute(statement, params)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 948, in execute
    return meth(self, multiparams, params)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 269, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1060, in _execute_clauseelement
    compiled_sql, distilled_params
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1200, in _execute_context
    context)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1413, in _handle_dbapi_exception
    exc_info
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 265, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 248, in reraise
    raise value.with_traceback(tb)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1193, in _execute_context
    context)
  File "/home/ge0/.virtualenvs/dummyweb/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 509, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.IntegrityError: (sqlite3.IntegrityError) UNIQUE constraint failed: user.email [SQL: 'INSERT INTO user (username, password, email) VALUES (?, ?, ?)'] [parameters: ('WhiteScale.1890', '***********************************', '**********@gmail.com')] (Background on this error at: http://sqlalche.me/e/gkpj)
```

I know that I should have coded better. Still, this is a good issue to get started. Can anyone help me up on that?

### PR DESCRIPTION
**Pull request by RiseNet-Web**
Add line 55 | 61 62 63
Search mail in database to resolved #2

### CODE REVIEW

File: src/dummyweb/__init__.py [hunk 1]
```diff
@@ -52,11 +52,14 @@ def subscribe():
         account.generate_tagname(request.form["login"])
         account.set_password(request.form["password"])
         account.email = request.form["email"]
-
-        db.session.add(account)
-        db.session.commit()
-        account.create_directory()
-        flash(f"Account {account.username} registered!", "info")
-        return redirect(url_for("subscribe"))
+        if (User.query.filter_by(email=account.email) is None):
```
Comment (Ge0) [inline src/dummyweb/__init__.py]: Remove the extra parenthesis please.

**Review by Ge0 — CHANGES_REQUESTED** [review]
Sounds good!

Just a minor change and I shall test & merge.

**Comment by RiseNet-Web** [comment]
Change commited

**Review by Ge0 — APPROVED** [review]

**Comment by Ge0** [comment]
Works like a charm. Thank you very much for you help. :smile:

**Pull request merged**